schema/all-spec.json

```diff
@@ -38553,20 +38945,19 @@
     },
     "additionalProperties" : false
   },
-  "AWS_ECS_Cluster_ClusterSetting" : {
+  "AWS_ECS_Cluster_ClusterSettings" : {
     "type" : "object",
-    "description" : "http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-cluster-clustersetting.html",
+    "description" : "http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-cluster-clustersettings.html",
```

Comment:
this doesn't seem correct:
http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-cluster-clustersetting.html
http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-cluster-clustersettings.html